controller/src/main/java/org/jboss/as/controller/SimpleListAttributeDefinition.java

```diff
@@ -167,12 +167,12 @@ public Builder setWrapXmlList(boolean wrap) {
         }
 
         public SimpleListAttributeDefinition build() {
-            if (attributeMarshaller == null) {
-                attributeMarshaller = AttributeMarshallers.getSimpleListMarshaller(wrapXmlList);
+            if (getAttributeMarshaller() == null) {
+                setAttributeMarshaller(AttributeMarshallers.getSimpleListMarshaller(wrapXmlList));
             }
             //todo add parser for SimpleListAttributeDefinition, for now no one is using it yet.
```

Comment:
Is that TODO still relevant?
If that's the case it should point to a WFCORE to track it

Reply:
It is no longer relevant, we should remove this comment